src/app/features/theme-drawer/theme-list.tsx

```diff
@@ -0,0 +1,17 @@
+import { Flex, FlexProps } from '@stacks/ui';
+import { SettingsSelectors } from '@tests/integration/settings.selectors';
+import { UserSelectedTheme } from '@app/common/theme-provider';
+import { ThemeListItem } from './theme-list-item';
+
+export const ThemeList = (props: FlexProps) => {
+  const themes: UserSelectedTheme[] = ['Light', 'Dark', 'System'];
+
+  return (
+    <Flex flexWrap="wrap" flexDirection="column" {...props}>
+      {themes.map(theme => (
+        <ThemeListItem key={theme} data-testid={SettingsSelectors.NetworkListItem} theme={theme} />
+      ))}
+      <div style={{ height: '2rem' }}></div>
```

Comment:
This is to add some padding and avoid the rounded border being clipped by the hover effect of the last item,

![image](https://user-images.githubusercontent.com/116000646/198679595-86f2dbe9-17f0-46e0-b4b3-609cc9bf0a34.png)

Reply:
Looked around at how `Stack` is being used elsewhere as I'm not yet familiar with it, best I was able to come up with is giving it a bottom padding,

```ts
<Stack paddingBottom="2rem">
   {/* List */}
</Stack>
```

Reply:
Yep, use is correct, though better here with the padding to use the named spacing strings like `loose`, `extra-loose`, etc.

Reply:
Why wouldn't we add a margin to the existing `Flex`?  

On `Stack`, I'd follow guidance on Chakra docs https://chakra-ui.com/docs/components/stack/usage. I'd reserve use for when there is: 
- More than 1 element
- That requires equal spacing between them